core/Utils.vala

```diff
@@ -720,5 +720,29 @@ namespace Maya.Util {
     public string get_hexa_color (Gdk.RGBA color) {
         return "#%02X%02X%02X".printf ((uint)(color.red*255), (uint)(color.green*255), (uint)(color.blue*255));
     }
+    
+    /*
+     * Handles Week Settings
+     */
+    
+    public bool show_weeks () {
+        if (GLib.SettingsSchemaSource.get_default ().lookup ("org.pantheon.desktop.wingpanel.indicators.datetime", false) != null) {
```

Comment:
Since you use this string 4 times it's probably a good idea to just make it a constant